JavaScript/js/descriptorDefinition.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * For displaying descriptions to the player (text about game that replaces itself)
+ */
+class Descriptor {
```

Comment:
Is this a descriptor or a displayer? To me this class changes any arbitrary DOM element text passed to it. Later in the code it is used as a descriptor and a namer type.